Autopilot/Libraries/Drivers/Src/stm32f7/UART.cpp

```diff
@@ -406,6 +406,25 @@ StatusCode UARTPort::resetDMA() {
 	return STATUS_CODE_OK;
 }
 
+StatusCode UARTPort::registerDMAReceiveCallback(void (*f)()) {
```

Comment:
good work on understanding wtf is going on and being able to fix it. I don't really know what's going on, but that's ok. Will revisit once we can test this driver on the hardware.